### ISSUE TITLE
AGO 2011-15

### ISSUE THREAD

**Issue by eclee25** [HopkinsIDD/cholera-mapping-pipeline#373]

**Comment by QLLZ** [comment]
Data pull:
HASH:  `6772ea9da4f43ddc3da11c494a573af2ca3dd3c8`
[config](https://github.com/HopkinsIDD/cholera-configs/tree/master/no_covariate_production_2023/2011_2015_country)

**Comment by QLLZ** [comment]
Model run:
HASH: `6772ea9da4f43ddc3da11c494a573af2ca3dd3c8`

**Comment by eclee25** [comment]
The model chains look like they have converged, except the variation in rho, w, and sd_w posterior samples seems to be increasing in later parts of the trace plot. In addition, while the mean annual incident case maps look okay and match the GAM input quite well, there does not seem to be good concordance between the mean annual incidence rate estimates and the corresponding GAM input. Subnational observations seem sparse, so I wonder if the model had difficulty allocating cases spatially.

Opinion: Model output files and data comparison report need further investigation.

**Comment by javierps** [comment]
I agree that the differences may be due to the low amount of sub-national data/ large parts of the country not being covered. This is probably also the reason behind slight convergence issues with rho and sd_w. I don't think we can do much about it.

What would you suggest looking for in the output files? @eclee25 

Given that validation and convergence are OK I be ok with validating the report.

**Comment by eclee25** [comment]
Moved to Approve column after second review

**Comment by QLLZ** [comment]
[Config](https://github.com/HopkinsIDD/cholera-configs/blob/master/July_2023_nocovar_production_runs/standard_runs/config_prd-jul2023_AGO_2011_2015.yml)

Data pull & Model run:

HASH: 211913e0eb7d570ed62c48bc1ae9ecd6671a6cbc

**Comment by QLLZ** [comment]
[country data report](https://github.com/HopkinsIDD/cholera-mapping-reports/blob/July_2023_nocovar_production_runs/country_data_reports/AGO_2011-01-01_2015-12-31_prd-jul2023_9210ea807d5a79e7c437c7ddbd6bbb48.country-data-report.html)

**Comment by eclee25** [comment]
Convergence issues with w, sd_w, and rho but other diagnostics look okay. Issues could be related to having few subnational observations. 

Opinion: Rerun with do_sd_w_mixture: FALSE to push sd_w to lower values

**Comment by QLLZ** [comment]
[Updated config](https://github.com/HopkinsIDD/cholera-configs/blob/master/July_2023_nocovar_production_runs/exception_runs/config_prd-jul2023_AGO_2011_2015.yml)

**Comment by QLLZ** [comment]
Data pull & model run:

**Comment by QLLZ** [comment]
[Updated country data report](https://github.com/HopkinsIDD/cholera-mapping-reports/blob/July_2023_nocovar_production_runs/country_data_reports/AGO_2011-01-01_2015-12-31_prd-jul2023_28d6498d634c72f0e4aeda5a8fe7124f.country-data-report.html)

**Comment by eclee25** [comment]
AGO traceplots are a bit improved without the mixture sd_w prior but I do wonder if a lower rho prior might help

**Comment by javierps** [comment]
Agreed. I think what could help is to put a stronger prior on std_dev_w, but not sure this will solve everything as there are large parts of the country without any data.

**Comment by eclee25** [comment]
ID-57 run is approved. For future runs, we need to have `do_sd_w_mixture: no
Approve ID-57

**Comment by QLLZ** [comment]
Data pull, model run:

HASH: `05f598b95b5d42ac0049963f8a0b7e04d6f1dbd7`

**Comment by javierps** [comment]
Issues with convergences in rho, std_w and ws OK. Rhats obs OK. Rhats output slight issues at adm 2 with all values < 1.1.

Suggestion: Accept.

**Comment by eclee25** [comment]
Approve
